app/controllers/users_controller.rb

```diff
@@ -41,7 +41,7 @@ def terms
   def save
     @title = t "users.new.title"
 
-    if params[:decline]
+    if params[:decline] || !params[:read_tou] || params[:read_tou] == "0"
```

Comment:
You don't actually need both tests here as "truthiness" means that `"0"` is false and hence the first test will be true...